job/service.go

```diff
@@ -67,6 +67,7 @@ type ReplayManager interface {
 	Init()
 	Replay(context.Context, models.ReplayRequest) (string, error)
 	GetReplay(uuid.UUID) (models.ReplaySpec, error)
+	GetReplayList(uuid.UUID) ([]models.ReplaySpec, error)
```

Comment:
whats uuid here?

Reply:
I think this is project id, on line number 69 because we are taking replay id, its fine to omit the argument name but because here we are taking project id, its not so obvious that is replay id or project id or something else? I think we should add named argument here.

Reply:
agree, updated